frontend/src/metabase/containers/NewItemMenu/NewItemMenu.tsx

```diff
@@ -1,36 +1,54 @@
 import { connect } from "react-redux";
+import { push } from "react-router-redux";
 import _ from "underscore";
-import { closeNavbar } from "metabase/redux/app";
+
 import NewItemMenu from "metabase/components/NewItemMenu";
+
 import Databases from "metabase/entities/databases";
+import Search from "metabase/entities/search";
+
+import { closeNavbar } from "metabase/redux/app";
 import {
   getHasDataAccess,
   getHasDatabaseWithJsonEngine,
   getHasNativeWrite,
+  getHasDatabaseWithActionsEnabled,
 } from "metabase/selectors/data";
-import { Database } from "metabase-types/api";
+
+import { Database, CollectionItem } from "metabase-types/api";
```

Comment:
I feel like lately, we've been using `import type` for types rather than importing them as values. I always do that, but not sure it's something we follow.